app/assets/stylesheets/member-facing/registration.scss

```diff
@@ -8,8 +8,29 @@
     margin-bottom: 20px;
   }
 
+  p {
+    padding-bottom: 10px;
```

Comment:
Did you look at the effect that this had on all the other `<p>` tags rendered to the user? why not use a class to be safe?